### ISSUE TITLE
Input section overlays terminal text

### ISSUE THREAD

**Issue by aaron-hall-7** [warpdotdev/Warp#2929]
### Describe the bug

Warp input overhangs terminal output when pushed to the top of the screen

### To Reproduce

1. Open warp
2. Type `git log`

You should see output similar to this
![image](https://user-images.githubusercontent.com/14354805/231567866-d3f28340-310c-41d5-9d3a-7c7a4ac38fd4.png)

### Expected behavior

There are two more lines that should be display and I would expect the output to look like the following image

![image](https://user-images.githubusercontent.com/14354805/231568430-c8cc9f32-502c-4530-a180-8c6e4c667a39.png)

### Operating System

MacOS

### OS Version

13.2.1

### Shell Version

zsh 5.8.1 (x86_64-apple-darwin22.0)

### Warp Version

v0.2023.04.04.08.03.stable_01

### Additional context

This seems to be a relatively new issue but I cannot say when exactly it started. I thought it was when I switched to Starship prompt, but I've disabled that and reverted to the original warp prompt and can see it there as well. 

I've also tested with a stock theme to rule out my theme as well as various fonts, font sizes, and line heights. Going to font size 10 with line height 1.0 makes it even worse

### Does this block you from using Warp daily?

No

### Is this a Warp specific issue? (i.e. does it happen in Terminal, iTerm, Kitty, etc.)

Yes, this I confirmed this only happens in Warp, not other terminals.

### Warp Internal (ignore): linear-label:b8107fdf-ba31-488d-b103-d271c89cac3e

None

**Comment by aldoyh** [comment]
Yep confirmed, also facing the same in the top of screen and it's too annoying 😃 

![Screenshot 2023-04-12 at 11 14 56 PM](https://user-images.githubusercontent.com/5215449/231575592-2bb07a7d-69e7-41bb-9c25-a7676c9b6f15.jpg)

**Comment by dannyneira** [comment]
Hey @aaron-hall-7, thanks for submitting! Closing as duplicate since this was reported here: https://github.com/warpdotdev/Warp/issues/2915

TLDR; There is a fix going out in tomorrow's release that should fix this issue, as a temp workaround, you can disable the Sticky Command Header in `Settings > Features > General`

![CleanShot 2023-04-12 at 14 26 02](https://user-images.githubusercontent.com/16809145/231576550-209b4202-98c8-40f3-8a0d-847f4a6fb6ea.png)

**Comment by aaron-hall-7** [comment]
> Hey @aaron-hall-7, thanks for submitting! Closing as duplicate since this was reported here: #2915
> 
> TLDR; There is a fix going out in tomorrow's release that should fix this issue, as a temp workaround, you can disable the Sticky Command Header in `Settings > Features > General`
> 
> ![CleanShot 2023-04-12 at 14 26 02](https://user-images.githubusercontent.com/16809145/231576550-209b4202-98c8-40f3-8a0d-847f4a6fb6ea.png)

I'm sorry that I didn't come across that when I searched for the issues. I appreciate the quick fix and the workaround!
